### DISCUSSION TITLE
Icona di stato personalizzata

### DISCUSSION THREAD

**Discussion by Jones1982** [indomus/forum#628] [HUB: Home Assistant]
Buonasera,
sto provando a personalizzare le icone di stato dei sensori porta (Zigbee Sonoff) sulla mia mappa (vedi sotto - quella tonda verde)
![Cattura](https://user-images.githubusercontent.com/57654443/129967433-da0256ce-cfa0-4836-9ab5-3011b04cc0b0.PNG)

Ho inserito in configuration.yaml quanto segue:
```
sensor:
  - platform: template
     sensors:
       template_test_balcone:
          friendly_name: 'Test > Balcone'
          value_template: >-
            {%- if is_state("binary_sensor.sonoff_a48000bc70", "on") -%}
              Aperto
            {%- elif is_state("binary_sensor.sonoff_a48000bc70", "unavailable") -%}
              Non Disponibile
            {%- elif is_state("binary_sensor.sonoff_a48000bc70", "off") -%}
              Chiuso
            {%- endif %}
          entity_picture_template: >-
            {% if is_state("binary_sensor.sonoff_a48000bc70", "on") %}
              /local/balconeaperto.gif
            {% elif is_state("binary_sensor.sonoff_a48000bc70", "unavailable") %}
              /local/nondisponibile.gif
            {% elif is_state("binary_sensor.sonoff_a48000bc70", "off") %}
              /local/balconechiuso.gif
            {% endif %}
```
però vorrei integrare anche le informazioni della batteria e lo storico, come per il sensore "classico" sonoff: 
![Cattura3](https://user-images.githubusercontent.com/57654443/129968128-aaa24da1-6f33-4a51-a3ee-1111453db053.PNG)
poiché attualmente ho solo queste info (nei dettagli del sensore personalizzato):
![Cattura2](https://user-images.githubusercontent.com/57654443/129968346-9aa9ea03-ed49-4099-9f20-6221d54d9f43.PNG)

Non avendo una entity dedicata x il valore della batteria del citato sensore, come posso fare?

Come sempre, **RINGRAZIO** chi mi illuminerà...!!!

N.b: I sensori sono connessi al gateway zigbee Sonoff, e configurati con l'integrazione hacs Sonoff Lan.

**Comment by snakuzzo** [comment]
Leggi la doc dei [Legacy Sensor](https://www.home-assistant.io/integrations/template/#legacy-sensor-configuration-format)

anche se, come leggerai, è consigliato l'utilizzo dei [Trigger-based template sensor](https://www.home-assistant.io/integrations/template/#configuration-variables)

P.S. Non ha importanza come stai integrando i device...si sta ragionando solo sui template
